### PR TITLE
feat(fms): complete navaid page

### LINKED ISSUE #8744: CDU: Selecting an ILS on Selected navaids causes page to crash
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-07-04T19:47:34+00:00",
    "ref": "refs/heads/master",
    "sha": "74e195a8ce4d0b1dbe7fe7dce046f7e6c4b44aff",
    "actor": "BlueberryKing",
    "event_name": "manual",
    "pretty_release_name": "master:74e195a8",
    "version": "a32nx-v0.12.0-dev.74e195a"
}
```

### Describe the bug

Selecting an ILS through the LSK4L on the selected navaid pages causes a soft crash
Console
Selected navaids

### Expected behavior

CDU page not to crash

### Steps to reproduce

Tune an ILS by identifier on the RAD NAV page or, perform a flight and let the FMS auto tune it for you.
Go to the selected navaid page(data -> position monitor -> SEL NAVAIDS) and press LSK4L.

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

bruno_pt99

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #8744 

## Summary of Changes

Implemented the rest of the NAVAID page, and added some additional mapping from MSFS data to support it better.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Setup a flightplan with an ILS departure runway so it auto-tunes. Go to DATA -> POSITION MONITOR -> SEL NAVAIDS and press the left LSK next to the ILS (LSK 4L). Ensure the NAVAID page shows with the ILS data. Try a couple of VOR and NDB navaids (you can just enter the idents in the cyan IDENT field to look at a different one).

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_NavaidPage.js [hunk 1]
```diff
@@ -48,19 +60,23 @@ class CDUNavaidPage {
...
+            } else if (facility.sectionCode === 1 && facility.subSectionCode === 0) { // navaid && vhf
+                CDUNavaidPage.renderVorDme(facility, template);
+            }
+
+            template[2][0] = `{cyan}${facility.ident}{end}`;
 
+            // 3L
             template[3][0] = '\xa0\xa0\xa0\xa0LAT/LONG';
-            template[4][0] = `${new LatLong(facility.location.lat, facility.location.long).toShortDegreeString()}[color]green`;
+            template[4][0] = `{green}${new LatLong(latLon.lat, latLon.long).toShortDegreeString()}{end}`;
```
Comment (BlueberryKing) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_NavaidPage.js:75]: I don't think `toShortDegreeString()` actually gives the correct format here. There should be a slash between lat and long.

Comment (tracernz) [reply fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_NavaidPage.js]: Good catch. Fixed on this and the waypoint page too.

**Review by BlueberryKing — APPROVED** [review]

**Comment by nebj00la** [comment]
Quality Assurance Trainee Report

Discord Username  : nebj00la
Object of testing : #8851
Aircraft          : A32NX
Tier of Testing   : 1
Date              : 27/08/2024

Testing Process: 

1. Configured flight from KBOS to KPVD.
2. Set departure as ILS runway 4R.
3. Confirmed  NAVAID page showed respective data.
4. Changed to PVD VOR using cyan IDENT field, confirmed the respective data.
5. Changed departure to ILS runway 27.
6. Confirmed NAVAID page showed respective data.

Testing Results: 
Passed

Media:

**Comment by alepouna** [comment]
Needs a changelog patch and LGTM too

**Comment by tracernz** [comment]
> Needs a changelog patch and LGTM too

Already has both ;).

**Pull request merged**